monkey/tests/unit_tests/monkey_island/cc/setup/test_server_setup.py

```diff
@@ -0,0 +1,90 @@
+from json import dumps
+from pathlib import Path
+
+import pytest
+
+import monkey_island.cc.setup.config_setup  # noqa: F401
+from monkey_island.cc.arg_parser import IslandCmdArgs
+from monkey_island.cc.server_setup import _extract_config
+from monkey_island.cc.server_utils.file_utils import is_windows_os
+from monkey_island.cc.setup.island_config_options import IslandConfigOptions
+
+BAD_JSON = '{"data_dir": "C:\\test\\test"'
+
+
+@pytest.fixture
+def cmd_server_config_path(tmpdir) -> Path:
```

Comment:
Use `tmp_path` instead of `tmpdir`. `tmp_path` is a `pathlib.Path` object. `tmpdir` is a `py.path.local` object.